src/classes/GE_GiftEntryController.cls

```diff
@@ -40,6 +40,10 @@ public with sharing class GE_GiftEntryController {
     public static final String PURCHASE_CALL_TIMEOUT_MESSAGE = System.Label.geErrorRequestTimedout;
     private static final String BOOLEAN_DATA_TYPE = 'BOOLEAN';
 
+    @AuraEnabled(cacheable=true)
+    public static Map<String, Integer> getGiftBatchTotalsBy(String batchId) {
+        return new GiftBatchService().getGiftBatchTotalsMapFor(batchId);
```

Comment:
Should we passing a `String` `Id` to an an argument of type `Id`. I'm sure it works but it just feels weird to me. Maybe `Id.valueOf(batchId)` to be safe?

Reply:
What do you think of casting to `Id`? Pushed up an update.